src/main/resources/init_db.sql

```diff
@@ -1,20 +1,48 @@
-CREATE SCHEMA IF NOT EXISTS `taxi` DEFAULT CHARACTER SET utf8;
-USE `taxi`;
+CREATE SCHEMA IF NOT EXISTS `taxi_service` DEFAULT CHARACTER SET utf8;
+USE `taxi_service`;
 
 DROP TABLE IF EXISTS `manufacturers`;
 CREATE TABLE `manufacturers` (
                                         `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                        `name` VARCHAR(225) NOT NULL,
-                                        `country` VARCHAR(225) NOT NULL,
+                                        `name` VARCHAR(225),
+                                        `country` VARCHAR(225),
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                        PRIMARY KEY (`id`));
+                                        PRIMARY KEY (`id`)
+                             );
 
 DROP TABLE IF EXISTS `drivers`;
 CREATE TABLE `drivers` (
                                   `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
-                                  `name` VARCHAR(225) NOT NULL,
-                                  `license_number` VARCHAR(225) NOT NULL,
+                                  `name` VARCHAR(225),
+                                  `license_number` VARCHAR(225),
                                   `is_deleted` TINYINT NOT NULL DEFAULT 0,
-                                  PRIMARY KEY (`id`),
-                                  UNIQUE INDEX `id_UNIQUE` (id ASC) VISIBLE,
-                                  UNIQUE INDEX `license_number_UNIQUE` (`license_number` ASC) VISIBLE);
+                                  PRIMARY KEY (`id`)
+                       );
+DROP TABLE IF EXISTS `cars`;
+CREATE TABLE `cars` (
+                           `id` BIGINT(11) NOT NULL AUTO_INCREMENT,
+                           `model` VARCHAR(225),
+                           `manufacturer_id` BIGINT (11),
+                           `is_deleted` TINYINT NOT NULL DEFAULT 0,
+                           PRIMARY KEY (`id`),
+                           CONSTRAINT `car_manufacturer_id`
+                                FOREIGN KEY (`manufacturer_id`)
+                                    REFERENCES `taxi_service`.`manufacturers` (`id`)
+                                    ON DELETE NO ACTION
+                                    ON UPDATE NO ACTION
+                    );
+DROP TABLE IF EXISTS `cars_drivers`;
```

Comment:
let's first drop tables in the correct order, and then create them - to avoid mistakes